api/krusty/helmchartinflationgenerator_test.go

```diff
@@ -684,6 +684,200 @@ spec:
 `)
 }
 
+func TestHelmChartInflationGeneratorForMultipleKubeVersions(t *testing.T) {
+	th := kusttest_test.MakeEnhancedHarnessWithTmpRoot(t)
+	defer th.Reset()
+	if err := th.ErrIfNoHelm(); err != nil {
+		t.Skip("skipping: " + err.Error())
+	}
+
+	copyValuesFilesTestChartsIntoHarness(t, th)
+
+	th.WriteK(th.GetRoot(), `
+namespace: default
+helmCharts:
```

Comment:
Looks like two helms generating one time.
So, I think I prefer generating one `helmCharts` two times in this case due to eliminate side effects.

Reply:
Sure, I think I did what you wanted! Let me know :+1: